src/generic/Poly.jl

```diff
@@ -183,6 +183,16 @@ function constant_coefficient(a::PolynomialElem)
    return coeff(a, 0)
 end
 
+@doc Markdown.doc"""
+    tail(a::Generic.PolynomialElem)
+
+Return the tail of the given polynomial, i.e. the polynomial without its
+leading term (if any)
```

Comment:
I would personally prefer to unify this docstring with that for `MPolyElem`, but otherwise:
```suggestion
leading term (if any).
```